usr/tgtd.c

```diff
@@ -899,6 +955,13 @@ int main(int argc, char **argv)
 	ep_handlers->ha_endpoints[1].ha_user_data = NULL;
 	ep_handlers->ha_count += 1;
 
+	ep_handlers->ha_endpoints[2].ha_http_method = POST;
```

Comment:
use ha_count instead of hard core values.

Reply:
Done